src/main/java/org/apache/rocketmq/dashboard/model/request/TopicConfigInfo.java

```diff
@@ -91,6 +94,14 @@ public void setOrder(boolean order) {
         this.order = order;
     }
 
+    public String getMessageType() {
+        return messageType;
+    }
+
+    public void setMessageType(TopicConfig topicConfig) {
+        this.messageType = topicConfig.getAttributes().getOrDefault("message.type", "");
```

Comment:
It is better to set NORMAL when the message type is empty,
WDYT

Reply:
Yes right that will be more verbose.